trillian_api.proto

```diff
@@ -169,7 +179,7 @@ message GetEntryAndProofResponse {
 // Clients cannot directly modify the log data via this API.
 service TrillianLog {
     // QueueLeaf adds a single leaf to the queue.
-    rpc QueueLeaf (QueueLeafRequest) returns (google.protobuf.Empty) {}
+    rpc QueueLeaf (QueueLeafRequest) returns (QueueLeafResponse) {}
```

Comment:
@Martin2112 We could add a request field to control the "response view", ie, a full view returns all fields, wheres a partial view would return only a few hand-picked fields. Omitting some parts of the response by default might be acceptable too (leaf_value / extra_data, maybe?)